qa/workunits/cephtool/test.sh

```diff
@@ -2490,7 +2490,7 @@ function test_mon_osd_erasure_code()
   ceph osd erasure-code-profile set fooprofile a=b c=d
   ceph osd erasure-code-profile set fooprofile a=b c=d
   expect_false ceph osd erasure-code-profile set fooprofile a=b c=d e=f
```

Comment:
This failure is pretty weird. The change should affect only the `ceph osd erasure-code-profile set --force` commands; those without `--force` should be failing (if the profiles differ) even in the past.

Do you have a teuthology.log maybe?

Also, I found two other cases in `test_rados_tool.sh`. I'm pushing updated commit because that.